core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
I think the simplest might be just changing each use of true/false for the mobile one to the opposite of what it currently is?